ServerCore/Pages/Teams/Edit.cshtml

```diff
@@ -23,7 +30,14 @@
             <input asp-for="Team.Password" class="hidden" />
             <div class="form-group">
                 <label asp-for="Team.Name" class="control-label">Team name</label>
-                <input asp-for="Team.Name" class="form-control" />
+                @if (canChangeTeamName)
+                {
+                    <input asp-for="Team.Name" class="form-control" />
+                }
+                else
+                {
+                    <input class="form-control" asp-for="Team.Name" value="@Model.Team.Name" readonly />
```

Comment:
instead of `readonly` do `disabled`

Reply:
When I make it disabled, the form doesn't get the property when it gets submitted. This is especially a problem because the field will then clear itself when there are validation errors